packages/client/hmi-client/src/components/model/petrinet/model-configurations/tera-stratified-matrix-preview.vue

```diff
@@ -1,6 +1,7 @@
 <template>
 	<section>
-		<section class="matrix">
+		<div v-if="matrix.length > 20 || matrix[0].length > 20">Matrix is {{ matrix.length }}x{{ matrix[0].length }}</div>
```

Comment:
The 20 is kind of arbitrary, we probably want to adjust it when we have better guidance on how people would interact with this matrices in terarium.

Reply:
nitpick - can we have this 20 stored in a `const maxDisplayDimension` (whatever var name you want Im not picky)

Reply:
done